define/types.go

```diff
@@ -30,7 +30,7 @@ const (
 	Package = "buildah"
 	// Version for the Package.  Bump version in contrib/rpm/buildah.spec
 	// too.
-	Version = "1.27.0"
```

Comment:
Don't we need a change in buildah.spec too?

Reply:
Yes removed here: https://github.com/containers/buildah/commit/198b4c360b84725fd13c81fc2984fb6b73da12aa